examples/profile/controls/test.rb

```diff
@@ -3,9 +3,9 @@
   impact 1.0
   title 'Security Group: No ingress access to CIDR block 0.0.0.0/0'
   desc 'Security Groups must not allow inbound access from anywhere'
-  
+
   Vpc.new(id: ENV['vpc_id']).security_groups.each do |security_group|
-    describe security_group do
```

Comment:
This is designed to be able to work as previously written. Where you having issues with this line specifically?